next-client/.example-env.local

```diff
@@ -1,2 +1,9 @@
 # How env works on NextJS: https://nextjs.org/docs/pages/building-your-application/configuring/environment-variables#default-environment-variables
-PIPELINE_EXPRESS_URL= # This is by default localhost:8080 on dev
\ No newline at end of file
+PIPELINE_EXPRESS_URL= # This is by default localhost:8080 on dev
+# Firebase keys below should be found on your firebase project. These are not sensitive and can be shared with the client.
```

Comment:
nice, is there a particular page which shows these? like "under main menu > project settings in your Firebase admin account"?